src/components/SwitchToggle.tsx

```diff
@@ -0,0 +1,29 @@
+import { Switch, SwitchProps, NAMED_COLORS } from '@ironfish/ui-kit'
```

Comment:
nit(non-blocking): I see where this was borrowed from, but just FYI, there's a method for theming chakra components so you almost never have to style things using selectors like we're doing here. I can update this when we add dark mode for this.